src/app/shared/shared/shared.module.ts

```diff
@@ -1,14 +1,13 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { LazyLoadDirective } from '../directives/lazy-load.directive';
 import { ButtonComponent } from './button/button.component';
 
 export const sharedSchema: NgModule = {
-  declarations: [ButtonComponent],
-  imports: [CommonModule]
+  declarations: [ButtonComponent, LazyLoadDirective],
+  imports: [CommonModule],
+  exports: [LazyLoadDirective]
 };
 
-@NgModule({
-  declarations: [ButtonComponent],
-  imports: [CommonModule]
-})
+@NgModule(sharedSchema)
```

Comment:
Ivy doesn't like the variable being used.  It will send a error in production build. That is why you see a copy of the modules.

Reply:
Gotcha, I'll switch it back